ExomeDepth/settings.py

```diff
@@ -66,3 +68,13 @@
 correlation = 0.98
 number_calls = [35,200]
 del_dup_ratio = [15,85] 
+
+reanalysis_dic = {
+    "XXY":["female", "XXYMaleAsFemale"],
+    "X": ["male", "XFemaleAsMale"],
+    "asFemale": ["female", "AsFemale"],
+    "asMale": ["male", "AsMale"]
```

Comment:
Maybe remove 'as' and double gender description?

```
XXYFemale
XMale
Female
Male
```

Reply:
Nope, this is all I could bargain for ;)